test/export/test_serialize.py

```diff
@@ -49,6 +49,12 @@
 from torch.testing._internal.torchbind_impls import init_torchbind_implementations
 
 
+try:
```

Comment:
should we only apply this checked-in artifact testing for a couple (< 10) artifacts? This seems like a lot of binaries being checked in.. not sure if pytorch is ok with this

Reply:
not sure, let me talk to Richard